tests/test_utils.py

```diff
@@ -83,3 +122,16 @@ def get_normalization_uniform_prior(
     acceptance_prob = posterior.get_leakage_correction(x=true_observation)
 
     return posterior_likelihood_unnorm, posterior_likelihood_norm, acceptance_prob
+
+
+def check_c2st(x: Tensor, y: Tensor, alg: str, tol: float = 0.1) -> None:
+    """Compute classification based two-sample test accuracy and assert it close to
+    chance."""
+
+    score = c2st(x, y)
+
+    print(f"c2st for {alg} is {score.item():.2f}.")
```

Comment:
We are still printing here.

Reply:
Yes, I realised that this is actually quite practical. I would otherwise repeat this print statement for every test that uses c2st.